src/black/const.py

```diff
@@ -1,4 +1,19 @@
+from enum import Enum
+from typing import Any
+
 DEFAULT_LINE_LENGTH = 88
 DEFAULT_EXCLUDES = r"/(\.direnv|\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|venv|\.svn|_build|buck-out|build|dist)/"  # noqa: B950
 DEFAULT_INCLUDES = r"(\.pyi?|\.ipynb)$"
 STDIN_PLACEHOLDER = "__BLACK_STDIN_FILENAME__"
+
+
+class OutputLevels(Enum):
```

Comment:
Minor thing, but the name could be improved if you also prefer something like `OutStyle` or `OutputLevel` if you're adamant about alluding to logging levels. Anyways a singular form always sounds nicer in enums and module names etc. at least to me.

Reply:
`OutputStyleLevel`?

Reply:
Perhaps not 😅

Reply:
I would go with `OutputLevel` then 👍🏻

Reply:
how about `LogLevel(s)?`?